tests/testthat/test_ttest_power.R

```diff
@@ -0,0 +1,19 @@
+context("test ttest power analysis")
+test_that("ttest power analysis density plot data generation", {
+  # Case where a_ratio is skewed.
+  mtcars %>% exp_ttest_power(a_ratio = 0.01, alternative = "two.sided", paired = FALSE)
+  # Case with paired test.
+  mtcars %>% exp_ttest_power(a_ratio = 0.5, paired=TRUE)
+  # Case with "less" alternative.
+  mtcars %>% exp_ttest_power(a_ratio = 0.5, alternative = "less")
+
+  res <- generate_ttest_density_data_for_power(0.2, 100, 100, 2, 100)
+
+  model_df <- exp_ttest_power(mtcars, a_ratio=0.5, d=0.2, sig.level=0.05, beta=0.2, alternative="two.sided", n_start=10, n_end=1000, n_step=10)
+  res <- model_df %>% tidy_rowwise(model, type="summary")
+  expect_equal(colnames(res), c("Probability of Type 1 Error","Probability of Type 2 Error","Power","Effect Size (Cohen's d)","Degree of Freedom","Required Sample Size"))
```

Comment:
Check the output values in addition to the column name if the output is stable and comparable.

Reply:
Done.